Overlays/Posix/Overlays/Android/cmake/post_add.cmake

```diff
@@ -13,6 +13,7 @@ if (WN_ANDROID_WRAPPER)
             COMMAND ${CMAKE_COMMAND}
                 -DCMAKE_TOOLCHAIN_FILE=${WN_ANDROID_TOOLCHAIN_FILE}
                 -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+                -DCMAKE_CXX_COMPILER_LAUNCHER=${CMAKE_CXX_COMPILER_LAUNCHER}
```

Comment:
Do we also need it to wrap the C compiler launcher?

Reply:
Nope, since we dont set it. (C files may not get cached, but thats like ... 2 files) could fix in the future.